docs/how-does-it-work.mdx

```diff
@@ -81,9 +171,11 @@ Some more details about replication:
 - **Write-Ahead Log.** The storage engine uses a write-ahead log to replicate data. It means that the storage engine writes data to the log first and then replicates it to the destination bucket. If the destination bucket is unavailable, the storage engine stores data in the log until the destination bucket becomes available again.
 - **Filtering.** You can filter records by labels when you replicate a bucket. This is useful when you need to replicate only a portion of the data.
 
-You can manage replications by using the [**Replication API**](./http-api/replication-api) or provision them with environment variables. See [**Configuration**](./configuration) for more details.
```

Comment:
I would remove the Replication API link and only keep the one about the guide to make sure people are going to the important page:

You can manage replications by using the Replication API or provision them with environment variables. See the [**Replication Guide**](./guides/data-replication) for more details.